meta/main.yml

```diff
@@ -2,115 +2,19 @@ galaxy_info:
   author: Calum MacRae
```

Comment:
@jaredledvina could this be how to change the owner in ansible galaxy?

Reply:
Hmmm I might actually be, I'll take a look. (For everyone's context, I'm trying to get this role to show up under the `sensu` author space). Since that will most likely cause people to have to change how they'll pull this role, it'd be a breaking change, so I'll work that out in a separate PR to make the story clean there.